packages/uicore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.78.0",
+  "version": "3.79.0",
```

Comment:
this can be a patch instead of a minor version change since there is no API change or addition of features.